meta-mel/recipes-core/busybox/busybox_%.bbappend

```diff
@@ -1,5 +1,7 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/busybox:"
 
+DEPENDS += "${@bb.utils.contains('INCOMPATIBLE_LICENSE', 'GPLv3', bb.utils.contains('IMAGE_FEATURES', 'security-selinux', 'libselinux', '', d), '', d)}"
```

Comment:
checking image features this way is not the way to go. Please see how we utilize feature-selinux overrides.

Reply:
I believe we don't have feature overrides in Elm release

Reply:
Ah my bad.... definitely aging!